modules/security_hub/main.tf

```
@@ -26,3 +26,11 @@ resource "aws_securityhub_standards_subscription" "default" {
   standards_arn = each.value
   depends_on    = [aws_securityhub_account.default]
 }
+
+resource "aws_sns_topic_subscription" "datadog-security" {
```

```suggestion
resource "aws_sns_topic_subscription" "datadog_security" {
```

We should use `_` in resource names.